interfaces/IOCDM.h

```diff
@@ -392,6 +392,22 @@ class EXTERNAL KeyId {
     ISession::KeyStatus _status;
 };
 
+// @stubgen:include <com/IIteratorType.h>
+// @json 1.0.0
+struct EXTERNAL IOCDM : virtual public Core::IUnknown {
```

Comment:
Is IOCDM complete? It does not follow the OCDM.json precisely? Also it's missing notifications.